dashboards/Insight/Prometheus_Exporter_Status.json

```diff
@@ -4610,10 +4724,13 @@
         "allFormat": "glob",
         "current": {
           "selected": false,
-          "text": "ip-10-178-0-240.us-east-2.compute.internal",
-          "value": "ip-10-178-0-240.us-east-2.compute.internal"
+          "text": "",
+          "value": ""
+        },
+        "datasource": {
+          "type": "prometheus",
+          "uid": "PA58DA793C7250F1B"
```

Comment:
@anton-bystrov is this a correct datasource?

Reply:
No. Sorry, I didn't notice it.

Reply:
@anton-bystrov could you please suggest correct changes or commit here?